app/diagnostics.go

```diff
@@ -241,17 +241,19 @@ func trackConfig() {
 	})
 
 	SendDiagnostic(TRACK_CONFIG_FILE, map[string]interface{}{
-		"enable_public_links": utils.Cfg.FileSettings.EnablePublicLink,
-		"driver_name":         utils.Cfg.FileSettings.DriverName,
-		"amazon_s3_ssl":       *utils.Cfg.FileSettings.AmazonS3SSL,
-		"amazon_s3_signv2":    *utils.Cfg.FileSettings.AmazonS3SignV2,
-		"thumbnail_width":     utils.Cfg.FileSettings.ThumbnailWidth,
-		"thumbnail_height":    utils.Cfg.FileSettings.ThumbnailHeight,
-		"preview_width":       utils.Cfg.FileSettings.PreviewWidth,
-		"preview_height":      utils.Cfg.FileSettings.PreviewHeight,
-		"profile_width":       utils.Cfg.FileSettings.ProfileWidth,
-		"profile_height":      utils.Cfg.FileSettings.ProfileHeight,
-		"max_file_size":       *utils.Cfg.FileSettings.MaxFileSize,
+
+		"enable_public_links": 		utils.Cfg.FileSettings.EnablePublicLink,
+		"driver_name":         		utils.Cfg.FileSettings.DriverName,
+		"amazon_s3_ssl":       		*utils.Cfg.FileSettings.AmazonS3SSL,
+		"amazon_s3_signv2":    		*utils.Cfg.FileSettings.AmazonS3SignV2,
+		"at_rest_encryption_key":	*utils.Cfg.FileSettings.AtRestEncryptKey,
```

Comment:
Is there a reason you added the key here? This data is sent back to us for tracking if these config settings are being used, I don't think you want to send us your encryption key